spec/controllers/api/v1/users/checkins_controller_spec.rb

```diff
@@ -152,115 +154,120 @@
 
     context 'with no page param given' do
       it 'fetches the most recent checkins (up to 30 checkins)' do
-        get :index, params: params
-        expect(res_hash.first['id']).to be device.checkins.first.id
+        get :index, params: params, format: "json"
```

Comment:
We need to define request format explicitly, if we use [respond_with](https://github.com/earlymarket/Coposition/pull/664/files#diff-a575a07f8a1226ac20eb83d2d054ccd3R19) from controllers.